docs/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to.md

```diff
@@ -0,0 +1,551 @@
+* [Utf8JsonWriter](#utf8jsonwriter)
+
+## Differences in default JsonSerializer behavior compared to Newtonsoft.Json
+
+<xref:System.Text.Json> is strict by default and avoids any guessing or interpretation on the caller's behalf, emphasizing deterministic behavior. The library is intentionally designed this way for performance and security. `Newtonsoft.Json` is flexible by default.
+
+### Case-insensitive deserialization 
+
+During deserialization, `Newtonsoft.Json` does case-insensitive property name matching by default. The <xref:System.Text.Json> default is case-sensitive, which gives better performance since it's doing an exact match. For information about how to do case-insensitive matching, see [Case-insensitive property matching](system-text-json-how-to.md#case-insensitive-property-matching).
+
+If you're using `System.Text.Json` indirectly by using ASP.NET Core, you don't need to do anything to get behavior like `Newtonsoft.Json`. ASP.NET Core specifies the case-insensitive setting when it uses `System.Text.Json`.
+
+### Comments
+
+During deserialization, `Newtonsoft.Json` ignores comments in the JSON by default. The <xref:System.Text.Json> default is to throw exceptions for comments because the [RFC 8259](https://tools.ietf.org/html/rfc8259) specification doesn't include them. For information about how to allow comments, see [Allow comments and trailing commas](system-text-json-how-to.md#allow-comments-and-trailing-commas).
+
+### Trailing commas
+
+During deserialization, `Newtonsoft.Json` ignores trailing commas by default. It also ignores multiple trailing commas (for example, `[{"Color":"Red"},{"Color":"Green"},,]`). The <xref:System.Text.Json> default is to throw exceptions for trailing commas because the [RFC 8259](https://tools.ietf.org/html/rfc8259) specification doesn't include them. For information about how to allow trailing commas, see [Allow comments and trailing commas](system-text-json-how-to.md#allow-comments-and-trailing-commas). There's no way to allow multiple trailing commas.
+
+### Property names
+
+During deserialization, `Newtonsoft.Json` accepts property names surrounded by double quotes, single quotes, or without quotes. For example:
+
+```json
+{
+  "name1": "value",
+  'name2': "value",
+  name3: "value"
+}
+```
+
+`System.Text.Json` only accepts names in double quotes because that format is required by the [RFC 8259](https://tools.ietf.org/html/rfc8259) specification and is the only format considered valid JSON..
+
+### String values
+
+During deserialization, `Newtonsoft.Json` accepts string values surrounded by double quotes or single quotes. For example:
+
+```json
+{
+  "name1": "value",
+  "name2': 'value'
+}
+```
+
+`System.Text.Json` only accepts strings in double quotes because that format is required by the [RFC 8259](https://tools.ietf.org/html/rfc8259) specification. A value enclosed in single quotes results in a [JsonException](xref:System.Text.Json.JsonException) with the following message:
```

Comment:
nit: There is a bit of redundant info here (at least conceptually). I would recommend merging the property names and string values section because there is quite a bit of overlap (they are all JSON strings). The difference in handling is just a subtle detail of Newtonsoft.Json so I don't think that warrants a separate section. Folks, imo, deal with JSON strings uniformly (as a concept), and hence there is little value in differentiating name vs value.